### PR TITLE
Add Amordiscos

### PR DESCRIPTION
**Pull request by sguinetti**
From https://amordiscostv.com/ website. Like Telemúsica, the country of this channel is Colombia.

### CODE REVIEW

**Review by Sphinxroot — DISMISSED** [review]

File: channels/co.m3u [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="Amordiscos.co" tvg-country="CO" tvg-language="Spanish" tvg-logo="https://amordiscostv.com/wp-content/uploads/2021/02/thumbnail_Logo-Amordiscos-600x600-alpha-150x150.png" group-title="Music",Amordiscos
+https://5b464b69d264e.streamlock.net:/Channels_live/ngrp:amordiscos_all/playlist.m3u8
```
Comment (freearhey) [inline channels/co.m3u:3]: image

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/co.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/co.m3u:3]: vlcsnap-2021-10-02-16h15m24s071

Comment (dct-infra) [reply channels/co.m3u:3]: Works for me.

**Comment by jkemp814** [comment]
snapshot

**Review by BellezaEmporium — DISMISSED** [review]

**Comment by LaneSh4d0w** [comment]
For me it's OK. Maybe streamtest didn't like it.

File: channels/co.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/co.m3u:3]: I've understood why. Let me check.

Comment (LaneSh4d0w) [reply channels/co.m3u]: @freearhey Test again

**Comment by LaneSh4d0w** [comment]
I think your test failed due to an issue linked to two dots without ports being added in @freearhey 

Try again, took them off.

image

**Review by freearhey — APPROVED** [review]

File: channels/co.m3u (hunk 1 above)
Comment (freearhey) [reply channels/co.m3u]: now works

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**